src/js/components/Meter/stories/Labelled.js

```diff
@@ -4,26 +4,30 @@ import { storiesOf } from '@storybook/react';
 import { Grommet, Box, Meter, Stack, Text } from 'grommet';
 import { grommet } from 'grommet/themes';
 
-const LabelledMeter = () => (
-  <Grommet theme={grommet}>
-    <Box align="center" pad="large">
-      <Stack anchor="center">
-        <Meter
-          type="circle"
-          background="light-2"
-          values={[{ value: 30 }]}
-          size="xsmall"
-          thickness="small"
-        />
-        <Box direction="row" align="center" pad={{ bottom: 'xsmall' }}>
-          <Text size="xlarge" weight="bold">
-            30
-          </Text>
-          <Text size="small">%</Text>
-        </Box>
-      </Stack>
-    </Box>
-  </Grommet>
-);
+function LabelledMeter() {
+  const meterValue = 30;
+
+  return (
+    <Grommet theme={grommet}>
+      <Box align="center" pad="large">
+        <Stack anchor="center">
+          <Meter
+            type="circle"
+            background="light-2"
+            values={[{ value: meterValue }]}
```

Comment:
You could store in a variable `[{ value: meterValue }]` to avoid unnecessary re-render.

Reply:
Same answer with above. LGTM